participants/eternal-dreamer-task2.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
Hi, @eternal-dreamer, You have created two files. This one is in the correct place but is empty. You have also created a participant.txt which was not required. You will have to move the content from that file to this one and delete the participant.txt to be able to merge this. You can see what I am talking about on clicking the `Files changed` button on the top of this PR.